insonmnia/marketplace/config.go

```diff
@@ -0,0 +1,23 @@
+package marketplace
+
+import "github.com/jinzhu/configor"
+
+type MarketplaceConfig struct {
+	ListenAddr string    `yaml:"address"`
+	Eth        EthConfig `required:"true" yaml:"ethereum"`
+}
+
+func NewConfig(path string) (*MarketplaceConfig, error) {
+	cfg := &MarketplaceConfig{}
+	err := configor.Load(cfg, path)
+	if err != nil {
+		return nil, err
+	}
+
+	return cfg, nil
+}
+
+type EthConfig struct {
```

Comment:
How many Eth configs do we have in our repo?

Reply:
okay, will do!